src/main/charts/bitbucket/Changelog.md

```diff
@@ -1,5 +1,17 @@
 # Change Log
 
+
+## 1.0.0
```

Comment:
I think it is worth calling out that this is the first supported version:

> This is the first officially supported version of the helm chart.

Or something like that.

Reply:
done